src/Exolnet/Test/DatabaseMigrators/SQLiteDatabaseMigrator.php

```diff
@@ -2,6 +2,7 @@
 
 use Artisan;
 use Illuminate\Filesystem\Filesystem;
+use Illuminate\Support\Facades\DB as DB;
```

Comment:
You shouldn't need the `as DB`.

Reply:
Indeed! Could you update this Simon? Thanks!

Reply:
I updated the code. Thanks